src/rt/sections_ldc.d

```diff
@@ -526,3 +516,209 @@ body
 
     return cast(immutable)result;
 }
+
+import core.sys.posix.pthread;
```

Comment:
All of these additions need to be guarded by a `version (OSX):`.

Reply:
Ah, yes of course. Done and pushed.